models/publications/base.py

```diff
@@ -22,9 +22,9 @@ class BasePublication(models.Model):
 
     year = models.IntegerField()
 
-    pages = models.CharField(
-        max_length=31,
+    pages = models.PositiveIntegerField(
```

Comment:
Afaik, pages are generally given as a range that can take various forms like dashed-ranges ('1-4'), comma-separated values ('1, 2') or even comma-separated combinations of both forms ('1, 3-4').

Reply:
Oh, I didn't think of that. This makes sense though.